### DISCUSSION TITLE
[Feature] quick change use button

### DISCUSSION THREAD

**Discussion by Luis-Lourenco** [immich-app/immich#4426] [Feature Request]
### The feature

I use for personal photos and work photos. 

Every time I have to logout from my personal account, log in to work account and vice versa. 

It’s nice to add like a switch user button Or a  login history. 

By the way, I use iOS app. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2919
